### ISSUE TITLE
Feature Level Combination with LDA

### ISSUE THREAD

**Issue by shiva1393** [mravanelli/pytorch-kaldi#171]
Hello Sir ,
I want to test different combinations of features with LDA projection. So i trained upto tri3 in kaldi, to train in pytorch-kaldi i need ark format. 
In kaldi at tri2  level train_lda_mllt.sh i found
est-lda --write-full-matrix=$dir/full.mat --dim=$dim $dir/0.mat $dir/lda.*.acc 
by using full.mat i have to take LDA projections?
or 
after lda_mllt we will get final.mat : feats="ark,s,cs:utils/subset_scp.pl --quiet $N $sdata/JOB/feats.scp | apply-cmvn $cmvn_opts --utt2spk=ark:$sdata/JOB/utt2spk scp:$sdata/JOB/cmvn.scp scp:- ark:- | splice-feats $splice_opts ark:- ark:- | transform-feats $dir/final.mat ark:- ark:- |"
 Which script i should use for LDA projection to give neural network input. Any script can you suggest. or pytorch-kaldi support LDA projection followed by DNN?
